<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDataSource</class>
 <widget class="QDialog" name="DialogDataSource">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据源</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>331</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="serial">
    <attribute name="title">
     <string>串口</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="serial_openClose">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>89</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
     <widget class="QWidget" name="serial_setting" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>141</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>3</y>
         <width>91</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QComboBox" name="serial_port">
          <property name="toolTip">
           <string>可自己填写。</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="serial_baudrate"/>
        </item>
        <item>
         <widget class="QComboBox" name="serial_parity"/>
        </item>
        <item>
         <widget class="QComboBox" name="serial_databits"/>
        </item>
        <item>
         <widget class="QComboBox" name="serial_stopbits"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="serial_checkCrc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>检查CRC</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbPrintTransmitPacket">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>显示发送的包</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbPrintReceivePacket">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>显示接收的包</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="file">
    <attribute name="title">
     <string>文件</string>
    </attribute>
    <widget class="QGroupBox" name="file_pathGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>9</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>文件路径</string>
     </property>
     <widget class="QPushButton" name="file_selectFile">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file_path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="file_optionGroup">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>96</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="title">
      <string>选项</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>时标周期(ms):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file_period">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>0009; </string>
      </property>
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="file_ctrlGroup">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>8</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>控制</string>
     </property>
     <widget class="QPushButton" name="file_startStop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QPushButton" name="file_pauseContinue">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serial_openClose</tabstop>
  <tabstop>serial_port</tabstop>
  <tabstop>serial_baudrate</tabstop>
  <tabstop>serial_parity</tabstop>
  <tabstop>serial_databits</tabstop>
  <tabstop>serial_stopbits</tabstop>
  <tabstop>serial_checkCrc</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>file_startStop</tabstop>
  <tabstop>file_selectFile</tabstop>
  <tabstop>file_period</tabstop>
  <tabstop>file_pauseContinue</tabstop>
  <tabstop>file_path</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
